WordPress/Classes/Stores/NoticeStore.swift

```diff
@@ -130,6 +143,10 @@ enum NoticeAction: Action {
     case clearWithTag(Notice.Tag)
     /// Removes all Notices except the current one.
     case empty
+    // Prevents the notices from showing up untill an unlock action.
+    case lock
+    // Show the missed notices.
+    case unlock
```

Comment:
If we're going ahead with this strategy, we should consider adding unit tests for these in [`NoticeStoreTests`](https://github.com/wordpress-mobile/WordPress-iOS/blob/fix/notice-overlap-for-site-icon/WordPress/WordPressTest/Classes/Stores/NoticeStoreTests.swift). 🙂